integration-tests/pom.xml

```diff
@@ -56,4 +56,17 @@
         <module>camel-core</module>
         <module>camel-salesforce</module>
     </modules>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.sonatype.plugins</groupId>
+                <artifactId>nexus-staging-maven-plugin</artifactId>
+                <version>${nexus-staging-maven-plugin.version}</version>
+                <configuration>
+                    <skipNexusStagingDeployMojo>true</skipNexusStagingDeployMojo>
```

Comment:
Do we also need to apply that to the gradle-it module?

Reply:
You are absolutely right, even worse, this module was not skipping the regular deployment.

Reply:
Fixed.